pkg/cmd/project/delete/delete.go

```diff
@@ -37,6 +37,10 @@ func NewCmdList(f factory.Factory) *cobra.Command {
 				return err
 			}
 
+			if len(args) == 0 {
```

Comment:
I'm not clear on what this block is about. Is it actually required?

Reply:
yes its required because under this we try to access `args[0]` and if you just run `octopus project delete` go will panic with an index out of bounds when trying to do so. There for we will never be able to reach the no prompt mode where we select a project to delete in interactive mode.